pkg/harfbuzz

```diff
@@ -21,7 +22,7 @@ freetype
 CPPFLAGS="-D_GNU_SOURCE" CFLAGS="$optcflags" CXXFLAGS="$optcflags" \
 LDFLAGS="$optldflags -Wl,-rpath-link=$butch_root_dir$butch_prefix/lib" \
   ./configure -C --prefix="$butch_prefix" --disable-nls $xconfflags \
-  --with-icu=no
+  --with-icu=yes
```

Comment:
why ?

Reply:
webkitgtk explicitly checks for harfbuzz-icu. maybe I could write a patch to atleast skip the check

Reply:
build log: https://0x0.st/sWSw.log

Reply:
yeah it's check explicitly, log: https://0x0.st/sWSY.log

Reply:
hmm after spending some thought on the issue, i guess we should make a second package harfbuzz-icu (copy of the current harfbuzz package with icu on), leave harfbuzz untouched and let webkitgtk depend on the former instead